clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -909,10 +910,12 @@ instr_summary_t::construct(void *dcontext, INOUT app_pc *pc, app_pc orig_pc,
     if (*pc == nullptr || !instr_valid(instr)) {
         return false;
     }
-    DO_VERBOSE(3, {
-        instr_set_translation(instr, orig_pc);
-        dr_print_instr(dcontext, STDOUT, instr, "");
-    });
+    DO_VERBOSE(3,
+               {
+                   instr_set_translation(instr, orig_pc);
+                   dr_print_instr(dcontext, STDOUT, instr, "");
+               },
+               verbosity);
```

Comment:
Seems easier to have verbosity ordered before the statement?

Reply:
The macro is used exactly in one place, so chose to delete the macro and write the explicit check - imho, easier to understand, and one less macro.